### PR TITLE
ADD cheatsheet to new branch

### PR DESCRIPTION
**Pull request by magh82**
-what, why the changes we make?
make changes for the formatting
add more command : to see the differences of the cheat sheet

### CODE REVIEW

File: git-cheatsheet-maggie.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
-pwd
+# command line
```
Comment (tjjjwxzq) [inline git-cheatsheet-maggie.md:1]: header woo hooo :smiley:

File: git-cheatsheet-maggie.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+- ls -a : list all
+- tab > allow to get the file path with the initial file name
+- git add
+- git status
+- git commit -m "-remark-"
+- git push origin <branch name>
+- git config
+- git remote url
+- git diff : compare working with history
+- git diff --cached : compare staging with history
```
Comment (shirlaine) [inline git-cheatsheet-maggie.md:16]: clear explanation! 😸

File: git-cheatsheet-maggie.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
-git remote url
+- ls -a : list all
+- tab > allow to get the file path with the initial file name
+- git add
+- git status
+- git commit -m "-remark-"
+- git push origin <branch name>
+- git config
+- git remote url
+- git diff : compare working with history
```
Comment (shirlaine) [inline git-cheatsheet-maggie.md:15]: 💃

File: git-cheatsheet-maggie.md [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
-ls -a
-tab > allow to get the file path with the initial file name
-git add
-git commit -m "remark "
-git push origin
-git config
-git remote url
+- ls -a : list all
+- tab > allow to get the file path with the initial file name
+- git add
```
Comment (tjjjwxzq) [inline git-cheatsheet-maggie.md:9]: HELLO

File: git-cheatsheet-maggie.md [hunk 5]
```diff
@@ -1,10 +1,16 @@
...
+
+pwd : go back to home
 cd ~/
 
-ls -a
-tab > allow to get the file path with the initial file name
-git add
-git commit -m "remark "
-git push origin
-git config
```
Comment (shirlaine) [inline git-cheatsheet-maggie.md:9]: 🍺

**Review by tjjjwxzq — CHANGES_REQUESTED** [review]
Good job! LGTM

**Review by shirlaine — APPROVED** [review]
Emojis are fun :)

**Pull request merged**